frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/graphtable/DiscoverAction.java

```diff
@@ -56,7 +56,8 @@ public void actionPerformed(AnActionEvent anActionEvent) {
                 GitVfsUtils.getMacheteFilePath(selectedRepository), project, getGraphTable(anActionEvent),
                 getBranchLayoutWriter(anActionEvent)),
             /* okButtonText */ getString("action.GitMachete.DiscoverAction.discovered-branch-tree-dialog.save-button-text"),
-            /* cancelButtonVisible */ true).show(), NON_MODAL));
+            /* cancelButtonVisible */ true,
+            /* hasBranchActionHints */ true).show(), NON_MODAL));
```

Comment:
Hmm it's a bit questionable if Discover dialog should have the tool tips... WDYT guys? @micpiotrowski @mkondratek

Reply:
Actually, if help dialog doesn't have tooltips because it hasn't a context menu, so maybe also discover dialog shouldn't have tooltips.
For now, I changed it in that way.